README.md

```diff
@@ -14,12 +14,17 @@ The Ordebook framework consist of five parts might by found in root folder.
 `spark-orderbook-sdk` Spark Orderbook SDK is a Rust library for the Orderbook contract interaction. Detailed information is in Readme.md.
 
 
-## Running Tests
+## Running All Tests
 
 ```
 forc build --release & cargo test --release
 ```
 
+### Running Fuzz Tests:
```

Comment:
Sometimes: these errors

---- functions::core::match_order_many::success::fuzz_match_order_many stdout ----
Error: transaction reverted: InsufficientBalance((328189159652, 362968087167)), receipts: [Call { id: 0000000000000000000000000000000000000000000000000000000000000000, to: d5d32b61679149f6f3d680b60f1bd40844d3987a80ec4fd41357fc156b9c173c, amount: 23868886, asset_id: 0000000000000000000000000000000000000000000000000000000000000000, gas: 86078, param1: 10480, param2: 10498, pc: 11704, is: 11704 }, LogData { id: d5d32b61679149f6f3d680b60f1bd40844d3987a80ec4fd41357fc156b9c173c, ra: 0, rb: 15329379498675066312, ptr: 67101395, len: 24, digest: f8989ac2c15dcdc80eaceddc10fe0f54aed1a79aef9e8b5009631adf62541d68, pc: 46648, is: 11704, data: Some(00000000000000000000004c69...) }, Revert { id: d5d32b61679149f6f3d680b60f1bd40844d3987a80ec4fd41357fc156b9c173c, ra: 18446744073709486080, pc: 46656, is: 11704 }, ScriptResult { result: Revert, gas_used: 82480 }]

---- functions::core::cancel_order::success::fuzz_buy_base stdout ----
Error: transaction reverted: ArithmeticOverflow, receipts: [Call { id: 0000000000000000000000000000000000000000000000000000000000000000, to: f049cd0ed68e42ae897de0dbce5a5f6138c70a6d14776b4d01f176a6479a9eb6, amount: 0, asset_id: 0000000000000000000000000000000000000000000000000000000000000000, gas: 99991708, param1: 10480, param2: 10507, pc: 11552, is: 11552 }, Panic { id: f049cd0ed68e42ae897de0dbce5a5f6138c70a6d14776b4d01f176a6479a9eb6, reason: PanicInstruction { reason: ArithmeticOverflow, instruction: MUL { dst: 0x10, lhs: 0x10, rhs: 0x11 } (bytes: 1b 41 04 40) }, pc: 67908, is: 11552, contract_id: None }, ScriptResult { result: Panic, gas_used: 4879 }]

Reply:
My last commit resolves this.